src/py/flwr/client/app.py

```diff
@@ -79,6 +79,8 @@
 
 """
 
+TRANSPORT_TYPES = ["grpc-bidi", "grpc-rere", "rest"]
```

Comment:
Should we maybe have these as constants so users can import them? I think its ok either way, though.

Reply:
Good idea, I'll make them constants in the next PR, but I removed the (for now) unused `grpc-rere`:

```suggestion
TRANSPORT_TYPES = ["grpc-bidi", "rest"]
```